joark/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/joark/mappers/InntektEndringAarsakMapper.kt

```diff
@@ -24,7 +24,7 @@ fun InntektEndringAarsak.stringValue(): String {
         is NyStillingsprosent -> "Ny stillingsprosent: fra $gjelderFra"
         is Permisjon -> typpe + ": " + liste.joinToString(transform = printPeriode())
         is Permittering -> typpe + ": " + liste.joinToString(transform = printPeriode())
-        is Tariffendring -> typpe + ": fra $gjelderFra"
+        is Tariffendring -> "$typpe: fra $gjelderFra"
```

Comment:
Kanskje gjøre det konsistent for alle dette gjelder i så fall

Reply:
e9, jeg så ikke at de andre linjene i farten. Rart at Intellij bare klaget på den ene 🤔

Reply:
Fikset. Gjorde listetransformasjonen litt enklere også.